src/renderer/components/+namespaces/namespace-select-filter.scss

```diff
@@ -33,7 +33,6 @@
       &::before, &::after {
         content: ' ';
         position: absolute;
-        z-index: 20;
```

Comment:
Shouldn't this to be replaced with something like:

```css
z-index: var(--z-index-menu);
```

Reply:
Works nice without any `z-index` specified.